projects/plugins/jetpack/extensions/blocks/subscriptions/view.scss

```
@@ -96,3 +98,32 @@
 		}
 	}
 }
+
+.wp-block-jetpack-subscriptions__newsletter-categories {
+	margin-top: 8px;
+	margin-bottom: 24px;
+	display: flex;
+	justify-content: center;
+	flex-wrap: wrap;
+	gap: 8px;
+
+	.wp-block-jetpack-subscriptions__newsletter-category {
+		padding: 6px 10px;
+		font-family: "SF Pro Text", -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen-Sans, Ubuntu, Cantarell, "Helvetica Neue", sans-serif;
+		font-size: 14px;
+		font-weight: 500;
+		line-height: 20px;
+		color: $gray-600;
+		border: 1px solid #dcdcde;
```

Could this be a `$gray-200`?

```suggestion
		border: 1px solid $gray-200;
```